rudder-web/src/main/webapp/javascript/rudder/angular/filters.js

```diff
@@ -200,8 +200,13 @@ app.controller('filterTagCtrl', function ($scope, $http, $location, $timeout, $r
     return !$scope.only.key && !$scope.only.value;
   }
 
+  // Add a grace periot before effectively do the search
+  var to = null;
   $scope.searchTree = function(treeId) {
-    $(treeId).jstree('searchtag', $scope.searchStr, $scope.tags, $scope.only);
+    if (to) { clearTimeout(to); }
```

Comment:
and here check that

> if to === undefined

Reply:
In fact you can remove the check, calling clearTimeout on undefined cause no problem